server/controllers/dashboard/stats.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+/**
+ * Dashboard Stats Controller 
+ *
+ * @description This controller contains all API for stats 
+ * displayed on the landing page (Dashboard) 
+ * 
+ * @requires 'q'
+ * @requires 'moment'
+ * @requires '../../lib/db'
+ */
+
+// requirements 
+const Q      = require('q');
+const moment = require('moment');
+const db     = require('../../lib/db');
+
+// expose to the API 
+exports.invoices = invoiceStat;
+exports.patients = patientStats;
+
+/** 
+ * @function invoiceStat
+ * 
+ * @description This function help to get statistical data about invoices 
+ */
+function invoiceStat(req, res, next) {
+  let params = req.query;
+  let bundle = {};
+
+  // date handler 
+  let date = params.date ? 
+    moment(params.date).format('YYYY-MM-DD').toString() : 
+    moment().format('YYYY-MM-DD').toString();
+
+  // cancelled transaction type
+  const CANCELED_TRANSACTION_TYPE = 10;
```

Comment:
It would be good in the future to have these all in one place so that the system can be updated to match the database records. This isn't a change that should happen now.